pyanaconda/packaging/dnfpayload.py

```diff
@@ -448,6 +448,12 @@ def _configure(self):
 
         conf.reposdir = REPO_DIRS
 
+        # Two reasons to turn this off:
+        # 1. Minimal installs don't want all the extras this brings in.
+        # 2. Installs aren't reproducible due to weak deps. failing silently.
```

Comment:
I think these are two sentences so it would be nice to capitalize the **F**ailing word.